### DISCUSSION TITLE
ESP14_SHRGB1C_01 working

### DISCUSSION THREAD

**Discussion by topogio** [sbidy/wiz_light#125] [General]
Wanted to let everyone know that the Philips Candle RGB wifi bulbs work great with this addon... Model: ESP14_SHRGB1C_01

I am able to use all functions of this bulb.  Dimming, RGB, effects & color temp.

**Comment by Jelboen** [comment]
I have some trouble with WiZ ESP14_SHRGB1C_01. It will show up, but i can’t change temperature / colors. I can change brightness and on/off. I tried both Yaml and UI setup. Is this still working?
